src/azul/__init__.py

```diff
@@ -1467,7 +1467,9 @@ def docker_images(self) -> dict[str, ImageSpec]:
         'linux/amd64'
     ]
 
-    waf_rate_rule_name = 'RateRule'
+    waf_rate_rule_name = 'rate_rule'
+
+    waf_expensive_rate_rule_name = 'expensive_rate_rule'
```

Comment:
In a previous review, I had requested these to be moved to [terraform/api_gateway.tf.json.template.py](https://github.com/DataBiosphere/azul/pull/5869/files#diff-bc096c76b960be5584081690f09bdcc8fd67afe4f9e69c2048e00dc3028cab7a).

Reply:
The variables `waf_rate_rule_name` and `waf_expensive_rate_rule_name ` are used by both `api_gateway.tf.json.template.py` and `cloudwatch.tf.json.template.py`, necessitating them to be defined outside of the terraform directory.